src/source/Ice/IceAgent.c

```diff
@@ -613,9 +676,10 @@ STATUS iceAgentStartGathering(PIceAgent pIceAgent)
         PROFILE_CALL_WITH_T_OBJ(CHK_STATUS(iceAgentInitSrflxCandidate(pIceAgent)), pIceAgent->iceAgentProfileDiagnostics.srflxCandidateSetUpTime,
                                 "Srflx candidates setup time");
     }
-
+#if 0
```

Comment:
Remove this section instead of #if block since we moved this to `AddConfig` now.